WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -822,6 +926,43 @@ - (void)controllerDidChangeContent:(NSFetchedResultsController *)controller
     [self.tableView reloadData];
     [self updateEditButton];
     [self maybeShowNUX];
+    [self maybeShowNoResultsView];
+}
+
+#pragma mark - WPNoResultsViewDelegate
+
+- (void)didTapNoResultsView:(WPNoResultsView *)noResultsView
+{
+    // Find the button in the NoResultsView
+    UIButton *button = nil;
+    for (UIView *subview in noResultsView.subviews) {
```

Comment:
Hmm, maybe we should go ahead and add a property for `button`? Rather relying on iterating the first level of subviews.